test/service/putGet.test.sh

```diff
@@ -0,0 +1,50 @@
+#!ic-repl -r http://localhost:8000
+
+// service address
+import S = "rwlgt-iiaaa-aaaaa-aaaaa-cai";
+
+identity Alice;
+
+call S.createProfile("alice", null);
+assert _ != (null : opt null);
```

Comment:
I think this is less verbose:  `assert _? == null`. No type annotation needed here, because `null` is actually of `null` type, not opt type.